src/main/scala/performanceanalysis/logreceiver/alert/AlertRuleActor.scala

```diff
@@ -19,15 +19,15 @@ class AlertRuleActor(alertingRule: AlertingRule, componentId: String, metricKey:
   lazy val actionActor: ActorRef = create(context)
 
   override def receive: Receive = {
-    case msg: CheckRuleBreak if doesBreakRule(msg.value) =>
+    case CheckRuleBreak(value) if value.isInstanceOf[Duration] && doesBreakRule(value.asInstanceOf[Duration]) =>
       log.info(s"Rule $alertingRule is broken for $componentId/$metricKey")
```

Comment:
`{}` will make sure the string is only build when it is actually logged. String interpolation is always evaluated when this statement is hit.